integrations/opencost/main.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
+from enum import StrEnum
+from port_ocean.context.ocean import ocean
+from client import OpenCostClient
+import asyncio
+
+
+class ObjectKind(StrEnum):
+    COST = "cost"
+
+
+def init_client() -> OpenCostClient:
+    return OpenCostClient(
+        ocean.integration_config["app_host"],
+        ocean.integration_config.get("window", "today"),
+    )
+
+
+async def process_cost_item(item: dict[str, Any]) -> list[dict[str, Any]]:
```

Comment:
The function is not supposed to be async, and then you can get rid of the `asyncio.gather`

Reply:
Noted. I've updated the PR